views/forms/self_referral_form.php

```diff
@@ -138,21 +141,20 @@
                                     <br>
                                     <div class="form-group row">
                                         <label class="col-form-label col-sm-2" for="self_pers_firstname">Participant Name:</label>
-
                                         <div class="col-sm-2">
-                                            <input type="text" class="form-control" id="self_pers_firstname" name="self_pers_firstname" placeholder="First name" required>
-                                            <div class="invalid-feedback">Enter first name</div>
+                                            <input type="text" class="form-control" id="self_pers_firstname" name="self_pers_firstname" placeholder="First name" oninput="validation()" required>
+                                            <div id="fname_error" class="invalid-feedback">Enter first name</div>
                                         </div>
 
                                         <label class="col-form-label col-sm-0 sr-only" for="self_pers_lastname">Last Name:</label>
                                         <div class="col-sm-2">
-                                            <input type="text" class="form-control" id="self_pers_lastname" name="self_pers_lastname" placeholder="Last name" required>
-                                            <div class="invalid-feedback">Enter last name</div>
+                                            <input type="text" class="form-control" id="self_pers_lastname" name="self_pers_lastname" placeholder="Last name" oninput="validation()" required>
```

Comment:
Don't need `oninput="validation()"` since you handle it in the new javascript function.